app/services/exception_hunter/error_creator.rb

```diff
@@ -23,6 +23,17 @@ def update_error_group(error_group, error)
 
         error_group.save!
       end
+
+      def include_user(**error_attrs)
+        user = error_attrs[:user]
+        unless user.nil?
+          error_attrs[:user_data] =
+            UserAttributesCollector.collect_attributes(user)
+        end
```

Comment:
```suggestion
        error_attrs[:user_data] = if user.nil?
                                    {}
                                  else
                                    UserAttributesCollector.collect_attributes(user)
                                  end
        end
```

Just so we don't have to do null checks later on. PS: you can't imagine how hard it was to write this code in Github 😰